### ISSUE TITLE
Animation tutorial "Follow mouse" example doesn't work in Sugar

### ISSUE THREAD

**Issue by icarito** [platipy/spyral#58]
"input.mouse.motion" event only seems to trigger on click+drag action.

I would guess this is a limitation of sugargame.

For latest minigame I had to resort to checking sugargame's PyGameCanvas get_pointer on every update.

**Comment by acbart** [comment]
That's pretty serious, if that's the case. We'll need to look into this, but I don't know when... Sorry again, its finals week for me.

**Comment by icarito** [comment]
No problem at all. I just want you to know I'm still super excited with 
Spyral and have already distributed it to the children of our workshop, 
in two different activities.

In case you need to prioritize, the startup delay is the most painful 
issue. We've "worked around it" by adding a splash screen, but 30+ secs 
for startup is quite excessive. Also the issue makes scenes practically 
unusable.

Hope all goes well in your finals!

**Comment by acbart** [comment]
I totally agree; we've identified the problem with that, and @rdeaton said he'd take a look at it tonight. We're about to do our own deployment soon, so I think it's critically important that we get it resolved soon. Otherwise I'm just going to have to completely remove all the styling functionality!

**Comment by icarito** [comment]
Here you can find the example I'm working on, Neko activity. Note it's embedded inside our authoring tool, Artisan Workshop: https://github.com/icarito/taller-artesano

**Comment by acbart** [comment]
Anyone know the status of this? Not sure if we managed to fix it?
